openlcs/reports/models.py

```diff
@@ -120,6 +127,13 @@ class CopyrightDetection(models.Model):
 
     class Meta:
         app_label = 'reports'
+        constraints = [
+            models.UniqueConstraint(
+                fields=['statement', 'false_positive', 'start_line',
+                        'end_line', 'false_positive', 'file_scan'],
```

Comment:
didn't verify if it makes sense to include these as unique. but you've added twice for `false_positive`.

Reply:
Thanks, updated it, I am still check if there exist a better way to set the uniqueconstraint